fast64_internal/oot/data/oot_actor_data.py

```diff
@@ -0,0 +1,43 @@
+class OoT_ActorData:
+    """Everything related to OoT Actors"""
+
+    from dataclasses import dataclass
+
+    @dataclass
+    class OoT_Actor:
+        id: str
+        key: str
+        name: str
+        category: str
+        tiedObjects: str
```

Comment:
this should be moved out of the class imo, it's unusual to define stuff inside classes, and this fits the module (oot_actor_data) anyway so okay to have at the global level